### ISSUE TITLE
[Quest] Fate, Up Against Your Will - Does Not Start

### ISSUE THREAD

**Issue by nefistis** [dalaranwow/dalaran-wow#5232]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: In Stormwind I turned in the quest "Reborn From The Ashes". Jaina Proudmoore was suppose to beside Varian but she is not. She is suppose to telport me to Orgrimmar. I am on Kirin Tor server.

**Expected behaviour**: She should be beside Varian when I turn in the quest so I can talk to her.

**Steps to reproduce the problem**:

1. Finish the "Reborn From The Ashes" Quest.
2. Talk to Varian

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
This works fine on PTR.

![GitHub Logo](https://i.imgur.com/Hq0hgcu.jpg)
![GitHub Logo](https://i.imgur.com/9r0uUua.jpg)
![GitHub Logo](https://i.imgur.com/VJCbkoE.jpg)
![GitHub Logo](https://i.imgur.com/NjM5ADx.jpg)

Jaina (guid 3106992) is in phase 1, has aura 60878. 

So you require ID - 60877 WGA Detect Invis (https://wowgaming.altervista.org/aowow/?spell=60877) which you will get, when you start quest 13347.
